src/mscorlib/src/System/Guid.cs

```diff
@@ -1014,6 +1024,34 @@ public byte[] ToByteArray()
             return g;
         }
```

Comment:
From a code reuse perspective, it'd be ideal if this method could just be:
```C#
var g = new byte[16];
TryWriteBytes(g);
return g;
```
but that will add some amount of overhead, so before making such a change, we'd want to understand what if any measurable impact there would be.